extensions/agroal/spi/src/main/java/io/quarkus/agroal/spi/JdbcUpdateSQLGeneratorBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.agroal.spi;
+
+import java.util.function.Supplier;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class JdbcUpdateSQLGeneratorBuildItem extends MultiBuildItem {
```

Comment:
Add some javadoc explaining what is this for, so it can be listed in https://quarkus.io/guides/all-builditems

Reply:
@gastaldi done